docs/requirements.txt

```diff
@@ -1,9 +1,19 @@
+pytest>=3.5
+h5py>=3.4, <4
+numpy>=1.20, <2
+scipy>=1.1, <2
+matplotlib>=3.5
+tifffile>=2020.9.30
+tabulate>=0.8.8, <0.9
+opencv-python-headless>=3.0
+cachetools>=3.1
+deprecated>=1.2.8
+scikit-learn>=0.18.0
+scikit-image>=0.17.2
+tqdm>=4.27.0
```

Comment:
Don't do this. Install the package instead (that'd be more similar to how it was previously handled and doesn't duplicate these dependencies). Hang on, I'll push a commit. If you agree, you can just squash it into the previous one (it also undoes the changes to the reqs).

Reply:
definitely much nicer to not do this. I could've sworn I tried what you added in your commit, but maybe just not that particular combination of everything. At any rate, thanks for figuring this out